zapcli/zap_helper.py

```diff
@@ -394,5 +394,5 @@ def html_report(self, file_path):
     @staticmethod
     def _write_report(report, file_path):
         """Write report to the given file path."""
-        with open(file_path, 'w') as f:
+        with open(file_path, encoding='utf-8', mode='w') as f:
```

Comment:
Hmm, I'm pretty sure the `encoding` parameter is only available in Python 3, and https://github.com/Grunny/zap-cli/issues/27 reports the issue in the Docker image which uses Python 2.7, so I don't think this would fix the issue in that case. Also, does this definitely happen in Python 3.5 (it's default handling of encoding is usually better)?

What might work for both Python 2.7 and Python 3 instead is opening in binary mode instead, i.e.
```python
with open(file_path, 'wb') as f:
```

But, I would need a test page that creates the issue to confirm. Do you know which part of the report is able to gain these characters? i.e. when you get the error, where is the character it reports the error for located in `http://zap/OTHER/core/other/htmlreport/?apikey=<YOUR API KEY>&formMethod=GET` ?